src/mixins/enum-mixin.ts

```diff
@@ -159,6 +154,16 @@ export default class EnumMixin extends Vue {
     return EnumUtilities.isTypeAdministrativeDissolution(item)
   }
 
+  /** DEPRECATED Returns True if filing is an Involuntary Dissolution. */
```

Comment:
I don't understand why you're adding deprecated code.

Reply:
To be consistent in how these methods are laid out. This will be all updated in ticket 15234.

The alternative is a "special case" for this method, which probably makes things a bit more complicated than being consistent.

Reply:
What would you think about replacing all calls to the `enum-mixin.ts` in `FilingHistory.vue` with calls to `enum-utilities.ts`?  I know it's slightly more work, but they're both pure functions and should be drop-in replacements for each other.  Once complete, we'd be one step closer to retiring the deprecated code in the `enum-mixin.ts`

Reply:
OK.

Reply:
Done. Please resolve this conversation if OK.